### PR TITLE
feat: init toilet details panel

### PR DESCRIPTION
**Pull request by stphnnnn**

### CODE REVIEW

File: src/components/ToiletDetailsPanel.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            </button>
+          </Box>
+
+          <Box>
+            {titleFragment}
+            {getDirectionsFragment}
+          </Box>
+
+          <Box>
+            <Text fontWeight="bold">Features</Text>
```
Comment (falconx) [inline src/components/ToiletDetailsPanel.js:163]: heading

File: src/components/ToiletDetailsPanel.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+              {Object.entries(features).map(([key, feature]) => (
+                <li key={key}>
+                  {feature.icon} {feature.label} {feature.valueIcon}
+                </li>
+              ))}
+            </ul>
+          </Box>
+
+          <Box>
+            <Text fontWeight="bold">Notes</Text>
```
Comment (falconx) [inline src/components/ToiletDetailsPanel.js:174]: heading

File: src/components/ToiletDetailsPanel.js [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+            <div>
+              {data.notes &&
+                data.notes
+                  .split('\n')
+                  .map((string, i) => <p key={i}>{string}</p>)}
+            </div>
+          </Box>
+
+          <Box>
+            <Text fontWeight="bold">Opening Times</Text>
```
Comment (falconx) [inline src/components/ToiletDetailsPanel.js:184]: heading

File: src/components/ToiletDetailsPanel.js [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+  faKey,
+  faCog,
+} from '@fortawesome/free-solid-svg-icons';
+
+import Box from './Box';
+import Button from './Button';
+import Text from './Text';
+import styled from '@emotion/styled';
+
+const CloseIcon = () => (
```
Comment (falconx) [inline src/components/ToiletDetailsPanel.js:23]: needs accessible label

File: src/components/ToiletDetailsPanel.js [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+    >
+      <Grid>
+        <Col gridColumn="1">{titleFragment}</Col>
+
+        <Col gridColumn="2">
+          <Box display="flex">
+            <Box mr={2}>
+              <FontAwesomeIcon icon={faClock} />
+            </Box>
+            <Text fontWeight="bold">Opening Hours</Text>
```
Comment (falconx) [inline src/components/ToiletDetailsPanel.js:209]: heading

File: src/components/Button.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      secondary: {
+        borderColor: 'primary',
+      },
+    },
+  })
+);
+
+const Button = ({ children, icon, ...props }) => (
+  <StyledButton type="button" {...props}>
+    <Text color="primary">{icon && <ButtonIcon icon={icon} />}</Text>
```
Comment (falconx) [inline src/components/Button.js:40]: should the condition also wrap `<Text color="primary">`?

Comment (stphnnnn) [reply src/components/Button.js:40]: definitely

**Pull request merged**